src/pudl/metadata/resources/allocate_net_gen.py

```diff
@@ -0,0 +1,196 @@
+"""Resource metadata for the allocate_net_gen tables."""
+from typing import Any
+
+RESOURCE_METADATA: dict[str, dict[str, Any]] = {
```

Comment:
instead of full duplicating each of these table everything could we do something like:
```
{f"table_name_{freq}_eia": {"description": f"{freq} table stuff", ...}  for freq in ["monthly", "annually"]}
```

Reply:
I'm not sure I understand what you're saying. Maybe we can chat?

Reply:
Okay I've applied this change to these resources and also the ones in the MCOE PR.